lte/gateway/python/scripts/generate_oai_config.py

```diff
@@ -148,35 +163,39 @@ def _get_attached_enodeb_tacs():
     return attached_enodeb_tacs
 
 
-def _get_apn_correction_map_list():
-    mme_config = load_service_mconfig("mme", MME())
-    if len(mme_config.apn_correction_map_list) == 0:
+def _get_apn_correction_map_list(service_config):
+    if len(service_config.apn_correction_map_list) == 0:
         return get_service_config_value("mme", "apn_correction_map_list", None)
-    return mme_config.apn_correction_map_list
+    return service_config.apn_correction_map_list
 
 
 def _get_context():
     """
     Create the context which has the interface IP and the OAI log level to use.
     """
+    mme_service_config = load_service_mconfig("mme", MME())
     context = {}
```

Comment:
Up to you, but creating this `context` might read a little cleaner if all the `_get_xxx` were inside the dict literal rather than as key-value writes